fdbserver/workloads/BlobGranuleCorrectnessWorkload.actor.cpp

```diff
@@ -911,7 +891,7 @@ struct BlobGranuleCorrectnessWorkload : TestWorkload {
 		loop {
 			state Transaction tr(cx, threadData->tenantName);
 			try {
-				Standalone<VectorRef<KeyRangeRef>> ranges = wait(tr.getBlobGranuleRanges(normalKeys));
+				Standalone<VectorRef<KeyRangeRef>> ranges = wait(tr.getBlobGranuleRanges(normalKeys, 1000000));
```

Comment:
Because we're now using the limit, we should assert here that the returned number of ranges is < 1000000, just to be explicit about the desired behavior

Reply:
Done.